index.js

```diff
@@ -8,6 +8,7 @@ class CircularDependencyPlugin {
     this.options = extend({
       exclude: new RegExp('$^'),
       failOnError: false,
+      allowAsyncCycles: true,
```

Comment:
I think the default should be `false` and consumers of this plugin can opt into ignoring these because they are in fact circular.

Reply:
Sure, I've changed it to false by default.